src/reference/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-bin.zip
```

Comment:
That's something is wrong. I don't believe that we need a separate Gradle just for docs...

Reply:
That was auto-generated.